### DISCUSSION TITLE
tlg-catalog/stable/tables/adverse-events/aet01_aesi

### DISCUSSION THREAD

**Discussion by giscus** [insightsengineering/tlg-catalog#90] [General]
# tlg-catalog/stable/tables/adverse-events/aet01_aesi

Safety Summary (Adverse Events of Special Interest)

https://insightsengineering.github.io/tlg-catalog/stable/tables/adverse-events/aet01_aesi.html

**Comment by staceyfu1029** [comment]
When I run the codes, the table shows the aesi_var but not their labels.

**Reply by shajoezhu** [reply]
hi @staceyfu1029, can you please check the var_labels of these column.

**Reply by staceyfu1029** [reply]
The var_labels is "Total number of patients with at least one" and the label of one of aesi_var FATAL is "AE with fatal outcome".

**Reply by shajoezhu** [reply]
Hi @staceyfu1029, can you share your code to reproduce this error? with the scda data. Thanks

**Reply by staceyfu1029** [reply]
Hi @shajoezhu, I found it was because my "tern" version was not up-to-date. Thank you for your help.
